src/main/java/games/strategy/engine/pbem/TripleAForumPoster.java

```diff
@@ -0,0 +1,202 @@
+package games.strategy.engine.pbem;
```

Comment:
Unit testing would be extremely hard, although not impossible... I will check if nodebb provides enough features to test posting without spamming the forum